src/proposal.mligo

```diff
@@ -448,4 +558,5 @@ let unfreeze (amt, config, store : unfreeze_param * config * storage) : return =
         ledger = ledger
       ; total_supply = total_supply
       ; freeze_history = new_freeze_history
+      ; period_frozen_history = update_period_frozen_history(int(amt), current_period, store.period_frozen_history)
```

Comment:
Note: this looks incorrect BTW, probably the it should be `- amt`

Reply:
This has been removed.